### PR TITLE
[Wallet] Fix revoked accounts appearing as verified

### LINKED ISSUE #5986: Revoked accounts appear to still be verified
### Expected Behavior
Revoked accounts are no longer considered to be verified.

### Current Behavior
Revoked accounts are considered to be verified.

Discussion:
https://celo-org.slack.com/archives/C018VLA3YAK/p1606250418219100
https://celo-org.slack.com/archives/C018VLA3YAK/p1606771248273600

### LINKED ISSUE #5987: Escrow withdrawals fail when user is paying for verification
### Expected Behavior
Escrow withdrawal is successful regardless of if the user has verified feelessly or not.

### Current Behavior
Escrow withdrawal is failing if the user has paid for their own verification.

### PR DESCRIPTION
**Pull request by tarikbellamine**
### Description
When an account is revoked using the `revoke` function, the account is no longer associated with the given identifier but the attestation stats for that account remain unchanged. In our current verification flow, we only check attestation stats when determining if a given account should be considered as verified. This was causing us to erroneously consider revoked accounts to still be verified.

This PR adds an explicit check to the verification flow for whether an account is associated with an identifier when determining if that account is verified. If the account is not associated with the expected identifier, it is not considered verified regardless of how many successful attestations it has.

Additionally, we now have a warning modal on the `revoke` button that tells users that revoked accounts will not be able to be reverified.

### Other changes
- Fixed a bug that was preventing escrow withdrawals to fail during user funded verification
- Added `bundle exec pod install` to the `build:wallet` script

### Tested
Yes

### Related issues
- Fixes #5986 
- Fixes #5987

### Backwards compatibility
Yes

### CODE REVIEW

File: packages/mobile/src/escrow/saga.ts [hunk 1]
```diff
@@ -267,6 +256,23 @@ function* withdrawFromEscrowUsingPepper(komenciActive: boolean = false) {
       select(feelessVerificationStateSelector),
     ])
 
+    if (!mtwAddress) {
+      yield call(withdrawFromEscrow)
+      return
+    }
```
Comment (jeanregisser) [inline packages/mobile/src/escrow/saga.ts:262]: Isn't this gonna fail anyway given there's no UI to enter the invite code to which the standard escrow is tied?

**Review by jeanregisser — COMMENTED** [review]
Thanks! 👍 
Could you add new tests to `verification.test.ts` covering this special case?

File: packages/mobile/src/escrow/saga.ts (hunk 1 above)
Comment (tarikbellamine) [reply packages/mobile/src/escrow/saga.ts:262]: Yeah good point. At least this way it will fail with the right error message.

**Review by jeanregisser — APPROVED** [review]
Awesome! Thanks for covering this with new tests 💯

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "reset-yarn": "yarn cache clean",
     "test": "yarn run lerna run test --concurrency 4",
     "build": "yarn run lerna run build --concurrency 4",
-    "build:wallet": "yarn build --scope @celo/mobile --include-filtered-dependencies",
+    "build:wallet": "yarn build --scope @celo/mobile --include-filtered-dependencies && cd packages/mobile/ios && bundle exec pod install && cd ../../..",
```
Comment (gnardini) [inline package.json:20]: This is really helpful, thanks!

**Pull request merged**